trackers/javascript-tracker/package.json

```diff
@@ -97,6 +97,8 @@
     "ts-node": "~10.7.0",
     "typescript": "~4.6.2",
     "wdio-chromedriver-service": "~7.3.2",
-    "webdriverio": "~7.19.3"
+    "webdriverio": "~7.19.3",
+    "@rollup/plugin-replace": "~4.0.0",
+    "chalk": "4.1.2"
```

Comment:
nit: I used to run package.json's through sort-package-json

`npx sort-package-json "**/package.json"`

I think I added it to the rush-prettier autoinstaller but I'm not sure I ever got it hooked up and working correctly.